intellij_codestyle_accuro.xml

```diff
@@ -47,12 +56,20 @@
     <option name="SPACE_WITHIN_SWITCH_PARENTHESES" value="true" />
     <option name="SPACE_WITHIN_SYNCHRONIZED_PARENTHESES" value="true" />
     <option name="SPACE_WITHIN_ARRAY_INITIALIZER_BRACES" value="true" />
+    <option name="SPACE_WITHIN_EMPTY_ARRAY_INITIALIZER_BRACES" value="true" />
+    <option name="SPACE_BEFORE_ARRAY_INITIALIZER_LBRACE" value="true" />
+    <option name="SPACE_BEFORE_ANNOTATION_ARRAY_INITIALIZER_LBRACE" value="true" />
+    <option name="CALL_PARAMETERS_WRAP" value="1" />
+    <option name="METHOD_CALL_CHAIN_WRAP" value="1" />
+    <option name="FOR_STATEMENT_LPAREN_ON_NEXT_LINE" value="true" />
```

Comment:
What effect does this have? It doesn't sound like something desirable but I can't see a difference when messing around in Settings

Reply:
When originally doing the style, I found a conversion guide online for Netbeans -> IntelliJ and managed to migrate these particular settings over. I will try and trigger them tomorrow and get back to you.

Thanks for noticing these; curious to find out myself.

Reply:
First option: 
    SPACE_BEFORE_ANNOTATION_ARRAY_INITIALIZER_LBRACE
    public boolean SPACE_BEFORE_ANNOTATION_ARRAY_INITIALIZER_LBRACE
    '@SuppressWarnings({"unchecked"}) or '@SuppressWarnings( {"unchecked"})

Second and third option determine how many params/method calls that are allowed beyond the margin.

See the following: 

![image](https://user-images.githubusercontent.com/2601492/74054349-bd683f00-4992-11ea-9b21-d082eccc1791.png)


FOR_STATEMENT_LPAREN_ON_NEXT_LINE is deprecated. So we can remove that.

@markvanrietschoten Any requests on the first three?

Reply:
@markvanrietschoten  built on that example a little better.

![image](https://user-images.githubusercontent.com/2601492/74054533-19cb5e80-4993-11ea-8ed5-53d9332331ce.png)

Reply:
@markvanrietschoten I've confirmed that the settings file will add a space at the right side end of the brace based on either inherited code settings, or something else at play.

The SPACE_BEFORE_ANNOTATION_ARRAY_INITIALIZER_LBRACE option is strictly for the left brace. There appears to be no option for after the array for the RBRACE, but the code style is currently doing what it should be with the option enabled.

Before
![image](https://user-images.githubusercontent.com/2601492/74061314-a03a6d00-49a0-11ea-9a9a-a459d7cfbe5a.png)

After
![image](https://user-images.githubusercontent.com/2601492/74061335-a892a800-49a0-11ea-9303-50f841483e6b.png)

Reply:
@TBRX103 I found https://github.com/JetBrains/intellij-community/blob/master/platform/lang-api/src/com/intellij/psi/codeStyle/CommonCodeStyleSettings.java which has simple examples of many settings, and yeah the annotation array init brace is fine. I think the general bracket/parenthesis spacing rules are otherwise correcting the spacing anyway, because my file which doesn't have that arg set explicitly still results in the 'After' example you give above after formatting. No harm in leaving it explicitly set anyway.

Reply:
@markvanrietschoten Awesome :) Thanks for looking into it as well